src/Apps/Order/__tests__/OrderApp.test.tsx

```diff
@@ -1,16 +1,145 @@
+import { routes } from "Apps/Order/routes"
 import { ContextProvider } from "Artsy/Router"
 import { mount } from "enzyme"
+import { Resolver } from "found-relay"
+import createRender from "found/lib/createRender"
+import getFarceResult from "found/lib/server/getFarceResult"
 import React from "react"
-import { HeadProvider } from "react-head"
-import { Meta } from "react-head"
+import { HeadProvider, Meta } from "react-head"
 import { ErrorPage } from "../../../Components/ErrorPage"
 import { OrderApp } from "../OrderApp"
 
+import { createMockNetworkLayer } from "Artsy/Relay/createMockNetworkLayer"
+import { Environment, RecordSource, Store } from "relay-runtime"
+
 jest.mock("react-stripe-elements", () => ({
   Elements: ({ children }) => children,
   StripeProvider: ({ children }) => children,
+  CardElement: () => jest.fn(),
+  injectStripe: () => jest.fn(),
 }))
 
+describe("OrderApp routing redirects", () => {
+  async function render(url, mockResolvers) {
+    const network = createMockNetworkLayer({
+      ...mockResolvers,
+    })
```

Comment:
`const network = createMockNetworkLayer(mockResolvers)`

Reply:
😅 evidence of "the faff"